docs/ManualMocks.md

```diff
@@ -181,3 +181,36 @@ the test file. But often you need to instruct Jest to use a mock before modules
 use it. For this reason, Jest will automatically hoist `jest.mock` calls to the
 top of the module (before any imports). To learn more about this and see it in
 action, see [this repo](https://github.com/kentcdodds/how-jest-mocking-works).
+
+### Mocking methods which are not implemented in JSDOM
+If some code uses a method which JSDOM (the DOM implementation used by Jest)
+hasn't implemented yet, testing it is not easily possible. This is e.g. the
+case with `window.matchMedia()`. Jest returns `TypeError: window.matchMedia is 
+not a function` and doesn't properly execute the test. 
+
+In this case, a manual mock in the test file should solve the issue:
```

Comment:
Nit: in Jest this is not a manual mock, "manual mocks" are providing a `__mocks__` and doing `jest.mock('./module/')`;